client/src/components/SelectedIngredientsList/styles.module.scss

```diff
@@ -0,0 +1,41 @@
+.selectedContainer {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  gap: 0.5rem;
+}
+
+.selectedIng {
+  text-transform: capitalize;
+  display: flex;
+  gap: 8px;
+  justify-content: flex-start;
+  padding-left: 1rem;
+  font-size: 1rem;
+  align-items: center;
+  background-color: var(--clr-primary-9);
+  height: 50px;
+}
+.icon {
+  font-size: 1rem;
+}
+
+@media (min-width: 600px) {
+  .selectedContainer {
+    grid-template-columns: repeat(3, 1fr);
+    gap: 1.25rem;
+  }
+}
+
+@media (min-width: 769px) {
+  .selectedContainer {
+    grid-template-columns: repeat(4, 1fr);
+  }
+}
+
+@media (min-width: 1025px) {
+  .selectedContainer {
+    grid-template-columns: repeat(4, 1fr);
+  }
+}
+@media (min-width: 1201px) {
+}
```

Comment:
This may be deleted as it is never used 😉

Reply:
Oo! Done :)